packages/e2e/spec/repositories-page.spec.ts

```diff
@@ -1,145 +1,116 @@
   });
 
-  test('should all be unchecked', async () => {
-    const repos = await page.repositoryList.all(2);
-    for (const repo of repos) {
-      await expect(repo.checkbox).not.toBeChecked();
-    }
+  test('should show an information box when no repositories are enabled', async () => {
+    expect(page.locator('sme-spatious-layout')).toContainText(
+      'It appears you do not have any enabled repositories, enable them by clicking on the enable repositories button.',
+    );
   });
 
-  test('should not show the modal dialog', async () => {
-    await expect(page.modalDialog.host).not.toExist();
-  });
-
-  test.describe('owner selector', () => {
-    test('should show the username and organization', async () => {
-      await expect(page.ownerSelector.options()).toHaveCount(2);
-      await expect(page.ownerSelector.options().nth(0)).toHaveAttribute(
-        'value',
-        'strykermutator-test-account',
-      );
-      await expect(page.ownerSelector.options().nth(1)).toHaveAttribute(
-        'value',
-        'stryker-mutator-test-organization',
-      );
+  test.describe('when opening the toggle dialog', () => {
+    test.beforeAll(async () => {
+      await openToggleDialog();
     });
 
-    test.describe('when selecting an organization', () => {
-      test.beforeAll(async () => {
-        await page.ownerSelector.select('stryker-mutator-test-organization');
-      });
-
-      test.afterAll(async () => {
-        await page.ownerSelector.select('strykermutator-test-account');
-      });
-
-      test("should show the repo's belonging to that organization", async () => {
-        await expect(page.repositoryList.repositoryNamesLocator).toContainText(
-          'github.com/stryker-mutator-test-organization/hello-org',
-        );
-      });
+    test('should open modal and show all available repositories', async () => {
+      expect(toggleModal).toContainText('Toggle repositories');
+      expect(await toggleModal.locator('sme-toggle-repository').count()).toBe(2);
     });
-  });
 
-  test.describe('when enabling a repository', () => {
-    test.beforeAll(async () => {
-      const repos = await page.repositoryList.all(2);
-      await repos[0].flipSwitch();
-    });
+    test('should enable repository', async () => {
+      const toggleRepository = toggleModal.locator('sme-toggle-repository').first();
+      await toggleRepository.locator('sme-toggle-button').click();
+      await toggleRepository.getByText('📋').click();
+      const text = await copyText();
 
-    test.afterAll(async () => {
-      if (await page.modalDialog.isVisible()) {
-        await page.modalDialog.close();
-      }
-    });
+      await closeToggleDialog();
+      const repository = page.locator('sme-repository');
 
-    test('should show the modal dialog', async () => {
-      await expect(page.modalDialog.host).toBeVisible();
-      await expect(page.modalDialog.title).toHaveText('hello-test');
+      expect(text.trim()).toMatch(API_KEY_REGEX);
```

Comment:
Is this `trim` necessary? it seems to me that we are only copying the API key.

Reply:
It was needed, but not anymore.